taqueria-vscode-extension/src/lib/gui/ContractsDataProvider.ts

```diff
@@ -23,7 +23,8 @@ export class ContractsDataProvider extends TaqueriaDataProviderBase
 			new ContractTreeItem(
 				path.basename(uri.path),
 				vscode.TreeItemCollapsibleState.None,
-				extensions[path.extname(uri.path).toLowerCase()],
+				smartContractLanguages.find(info => info.fileExtensions.indexOf(path.extname(uri.path).toLowerCase()) !== -1)
```

Comment:
What does this line do?

Reply:
This finds the `SmartContractLanguageInfo` that matches the file extension of the current contract, and sets the language name to be shown in the TreeViewItem

Reply:
I think it's better to add a few helper functions to simplify these.